subsystem/utils_test.go

```diff
@@ -155,6 +162,25 @@ func generateNTPPostStepReply(ctx context.Context, h *models.Host, ntpSources []
 	Expect(err).ShouldNot(HaveOccurred())
 }
 
+func generateApiVipPostStepReply(ctx context.Context, h *models.Host, success bool) {
```

Comment:
Why duplicate code it it's already exist in other tests? 
If you define a generic function then please fix the other tests to use it as well and put it in utils_test.go

Reply:
Fixed.